### ISSUE TITLE
WP 1.0 Bot stops & not create logs

### ISSUE THREAD

**Issue by AfroThundr3007730** [openzim/wp1#7]
From phabricator [T207599](https://phabricator.wikimedia.org/T207599):
>''WP 1.0 bot'' is having ongoing issues for several months. It starts daily processing, then stalls out at various WikiProjects, not processing the rest. Lately, it does not get past "B" in the alphabet. The multiple issue reports are at [Wikipedia talk:Version 1.0 Editorial Team/Index](https://en.wikipedia.org/wiki/Wikipedia_talk:Version_1.0_Editorial_Team/Index).
>
>A second issue is that the bot not creating WP logs for many WP since October 8. For example [Wikipedia:Version 1.0 Editorial Team/Catholicism articles by quality log](https://en.wikipedia.org/wiki/Wikipedia:Version_1.0_Editorial_Team/Catholicism_articles_by_quality_log)
and [Wikipedia:Version 1.0 Editorial Team/Saints articles by quality log](https://en.wikipedia.org/wiki/Wikipedia:Version_1.0_Editorial_Team/Saints_articles_by_quality_log).

**Comment by kelson42** [comment]
@audiodude Do you think this is still a current problem? What should we do with this ticket?

**Comment by audiodude** [comment]
Yes definitely still a problem. I believe a wikipedia admin protected the logging tables so that the current bot can't even attempt to update them.

**Comment by audiodude** [comment]
In my opinion #3 , #9 and this issue all represent the same underlying problem. They should be merged or all closed in favor of opening a new task. There is no interesting information in any of these bugs, they all just say "logs aren't working" in one form or another.

**Comment by AfroThundr3007730** [comment]
Every log page linked from the index were protected on Dec 6 [citing bot issues](https://en.wikipedia.org/wiki/Special:Log?type=protect&user=Gonzo+fan2007&page=&wpdate=2018-12-07&tagfilter=&subtype=) which are now buried in an archive. If the issues with the bot have been fixed, we could simply ping Gonzo fan2007 about unprotecting all those pages

**Comment by audiodude** [comment]
Unfortunately, to our knowledge, the issues are not fixed.

**Comment by kelson42** [comment]
@audiodude What kind of information about the underlying problem do you have? Why do you believe that having rewritting the WP1 bot collector part does not fix the problem?

**Comment by kelson42** [comment]
We will have to rewrite the upload part of the WP1 bot to fix bugs related to that log. Work on this milestone will start soon. Then we could request the page protection from these log page.

**Comment by kelson42** [comment]
FYI https://en.wikipedia.org/wiki/Wikipedia_talk:Version_1.0_Editorial_Team/Index#Logs_will_be_back_soon!

**Comment by kelson42** [comment]
@audiodude Maybe we could close that ticket now?

**Comment by audiodude** [comment]
I haven't merged the log creation code yet, because I was waiting for the p0 update/upload bugs to be fixed. There's also #72 which I'd like to investigate before we start updating logs.
